widgets/cpu.go

```diff
@@ -18,14 +18,17 @@ type CpuCoreStats struct {
 }
 
 type CpuStats struct {
-	cores int
-	stats []CpuCoreStats
+	cores      int
+	stats      []CpuCoreStats
+	totalStats CpuCoreStats
 }
 
 type CpuWidget struct {
-	cpuStats CpuStats
-	data     [][]float64
-	plot     *tWidgets.Plot
+	cpuStats     CpuStats
+	totalData    [][]float64
+	data         [][]float64
+	showEachCore bool
+	plot         *tWidgets.Plot
 }
 
 var HORIZONTAL_SCALE = 3
```

Comment:
```suggestion
var HorizontalScale = 3
```

저도 Intellij 인텔리센스때문에 알게 되었는데 camel-case로 상수를 표현한다고 하네요 [링크](https://stackoverflow.com/a/22688926)

Reply:
(사무실에서 얘기했던 것) Camel case 중에서도 첫 글자가 대문자면 upper camel case 라고 한다고 합니다.